### ISSUE TITLE
Update dokuwiki and php to Version 5.6.31

### ISSUE THREAD

**Issue by kkuka** [cosmocode/edittable#147]
Hi guys,
I've a problem since i have updated my dokuwiki and php to the latest version. Since this update the plugin editable will not work anymore. A click on the edittable button in the site editor has no affect (nothing happened).
I'm using a Redhat 6.9 (x86_64) and a Apache 2.2.15 from the Redhat repositories. PHP is installed in version 5.6.31 from the remi repos. 

I hope you can assist!

Kai

**Comment by kkuka** [comment]
Ah sorry. Here are some additional Infos:
Editable is viewable in the extension manager without any errors. 
I've forget to say that I made a few steps to fix this issue
- Try reinistall edittable via the extension manager (No success)
- Try to uninstall and then install edittable via the extension manager (No success)
- And finaly I try to uninstall and install it manualy (Also no success)

**Comment by micgro42** [comment]
Are you using dokuwiki stable (i.e. the Frusterick Manners release) or the dokuwiki master branch? The latter has a (almost fixed) bug splitbrain/dokuwiki#2115 that may interfere with edittable.

**Comment by kkuka** [comment]
Hi,
Yes. We're using the Frusterick Manners release. I'll give it a try with a newer release. 
I'll keep you informed.

**Comment by kkuka** [comment]
:-( 
Sorry. Same Host with a fresh new dokuwiki installation. Nothing changes!  I'll click on the icon and nothing happens.  Finally i'll try it on a fresh Redhat 7.

Regards,

**Comment by micgro42** [comment]
I don't think a new Redhat makes a difference unless you did strange things to your Apache or PHP config.
Could you have a look at your Apache error logs? They should be at `/var/log/httpd/`

**Comment by kkuka** [comment]
Hi,
Sorry for my late reply. But i needed time to test some setups.
I'm going crazy!!! This issue is reproducible!
I've made a few blank and fresh redhat 7 (php 7.1 from the remi repo) and ubuntu 16.04.3 (php 7.0) installations. All installations based on the minimalistics setups incl. the group Basic Web Server for redhat and LAMP for ubuntu. I made the test with the stable (2017-02-19e "Frusterick Manners") and the development (snapshot? update version 50) version of dokuwiki.
Edittable was installed by using the Extension Manager and manually (with different versions). The php.ini was always untouched!
Every time I got the same affect. Clicking on the edittable icon has no affect.

But, when I use dokuwiki version 2016-06-26e "Elenor of Tsort" – everything is fine…
Do I have forgotten to set any switches or parameter?!?!?

Strange …

**Comment by kkuka** [comment]
Hi again

I've made some further investigations and check your edittable against different Dokuwiki versions. And voila with dokuwiki-2017-02-19e (and newer) edittable will not longer work. 
With dokuwiki-2017-02-19d is everthing fine.

**Comment by micgro42** [comment]
This is strange. But thank you for pointing it out. I will look into it.

**Comment by micgro42** [comment]
Do you happen to have `<code>` blocks on your page? If so, does this happen to tables on pages without `<code>` blocks as well?

The only commit between those two versions is https://github.com/splitbrain/dokuwiki/commit/307920406bb9328abbe8aeaf737594bd2402ac94 and that shouldn't affect anything but code blocks. o.O

**Comment by kkuka** [comment]
I made the tests with a naked dokuwiki setup. Only the start page was configured with a simple table.
Btw. other plugins like dw2pdf and the LibreOffice export makes also trouble (The icons are not shown in the default theme). It looks like an issue with dokuwiki

**Comment by splitbrain** [comment]
@kkuka have you tried different browsers, yet? Made sure no adblocker is interfering? Checked the JavaScript console for any errors?

**Comment by kkuka** [comment]
@splitbrain Sure. I've tested it with different browser (FF 55, Chromium 61, IE8), OS's and webserver (Apache and nginx). All have the same affect
![screenshot - 29 09 2017 17_05_46](https://user-images.githubusercontent.com/31959501/31022270-865df5aa-a538-11e7-9a9b-40f139c710c2.png)
![screenshot - 29 09 2017 17_04_18](https://user-images.githubusercontent.com/31959501/31022286-8e6c6d26-a538-11e7-9e97-c90b5776a8b9.png)

**Comment by micgro42** [comment]
Could you disable the datatables plugin and see if it works then? Also, your screenshots show, that you are currently on Master. Edittable is known to be broken there. Could you go back to the most recent stable version of DokuWiki?

**Comment by kkuka** [comment]
I have the datatables plugin disabled, nothing changes in **2017-02-19e "Frusterick Manners"**. 
What do you mean with, that I am on Master? Is that a special release?

**Comment by micgro42** [comment]
No, you are not using 2017-02-19e "Frusterick Manners". The icons on the right of your screenshots indicate that you are using the current development build of DokuWiki, i.e. the Master branch from https://github.com/splibtrain/dokuwiki

Please go back to 2017-02-19e "Frusterick Manners", and make the first screenshot again with the datatables plugin disabled. Then we can see if anything is broken :-)
